client-proxy/database/database.go

```diff
@@ -0,0 +1,45 @@
+package database
+
+import (
+	"database/sql"
+	"github.com/OpenLineage/OpenLineage/client-proxy/config"
+	_ "github.com/mattn/go-sqlite3"
+	"log"
+)
+
+type Database struct {
+	db *sql.DB
+}
+
+type IDatabase interface {
+}
+
+func New(conf config.Config) *Database {
+	db, err := sql.Open("sqlite3", conf.SqliteDatabasePath)
+	if err != nil {
+		log.Fatalln("Cannot open database:", err)
+	}
+
+	log.Println("Initializing DB")
+	InitDB(db)
+	return &Database{db: db}
+}
+
+func InitDB(db *sql.DB) {
+	db.Exec(`
+		CREATE TABLE IF NOT EXISTS partitions (
+			name       text     NOT NULL,
+			size       integer  NOT NULL,
+			created_at datetime NOT NULL,
+			current    boolean  NOT NULL
+		)
+	`)
+
+	db.Exec(`
```

Comment:
Do we have a way to manage db migrations?

Reply:
Yes, there is a few library options that are ready to use. From this PR, I just did `db.Exec` just to verify the query itself and for some feedback regarding the column names, types, and etc. Evaluation and adoption of migration library itself could be a separate story, and I will address it from another PR.

Reply:
You may want to level a note, but otherwise adding / discussing migration support in a separate PR make sense